### ISSUE TITLE
I can't connect to the home page

### ISSUE THREAD

**Issue by JesseWhiteMaan** [AdrienPoupa/docker-compose-nas#26]
Hello,

I followed all the procedures, my docker is active, however when I type my IP to access the services if I leave http://mon-ip I get the error 404 page not found, if I want to access Sonarr etc, it turns in the void.

My docker runs on Ubuntu server, my client is on w10 with firewall disabled...

Do you have any clues?

Thanks

**Comment by JesseWhiteMaan** [comment]
Ok... Just a problem with .env. Nvm

**Comment by IdiosApps** [comment]
What was the problem with your .env? 
I'm unable to access any service. `http://localhost:8080/` (qbittorrent webui port), `http://localhost:8989/sonarr`, `http://vpn:8080/`, etc.

**Comment by AdrienPoupa** [comment]
@IdiosApps This is normal, Traefik proxies all of those to port 443.

qBittorrent will be accessible at https://localhost/qbittorrent, Sonarr at https://localhost/sonarr, etc. This is documented [here](https://github.com/AdrienPoupa/docker-compose-nas#applications).

Also your `.env` file should specify the hostname, if you are not using a domain name then you should have `HOSTNAME=localhost`, that's the case by default.

**Comment by IdiosApps** [comment]
@AdrienPoupa Right! I was browsing the docker-compose config and just figured that out. I've done some docker + simple networking with ports before, but not as far as Traefik etc. I managed to get to the homepage on https://localhost. 

I'm used to seeing e.g. `localhost:8080`: no http/https prefix, and some port postfix. Port 80/443 making it be http**s** (with no port postfix) makes sense now. `localhost:80` and `localhost:443` both redirect to `https://localhost`.

---

The readme has just one reference to `localhost`. I don't know if most users have non-default settings - but it could be handy to have at least one link to https://localhost(/app-name)/. If users can easily get to the homepage then they'll discover the other links.

In retrospect it's simple that I should just go to `localhost/app-name`. As a new user somehow it wasn't obvious though 🤣

**Comment by VMichailidis** [comment]
Hello, I have the same issue however I can't seem to the homepage.
My client tries to connect to 192.168.1.26 which is the output of hostname -I, but I get a connection privacy error and then a 404.
I have opened ports 80, 433 and 3000 via ufw.
For the variable HOSTNAME I have tried using both the value "localhost" and the value "heaviside" (the hostname I have manually assigned to my server during installation) and neither seem to work.
